### PR TITLE
Fixed the path issues

### PR DESCRIPTION
**Pull request by abishekrs24**

### CODE REVIEW

**Review by Javasamurai — APPROVED** [review]

**Review by vivekb91 — APPROVED** [review]

**Comment by rampr** [comment]
They are dependent on docs being presented at gcdoc/. These changes are fine for now.

**Review by rampr — APPROVED** [review]

**Review by Javasamurai — APPROVED** [review]

**Comment by rampr** [comment]
As discussed, these changes may not be correct

**Review by codesandeep — APPROVED** [review]

File: html/index.html [hunk 1]
```diff
@@ -37,12 +37,12 @@
 
           <div class="nav-collapse collapse">
             <ul class="nav pull-right">
-              <li><a href="/index.html#quickstart">Quick Start</a></li>
-              <li><a href="/index.html#guides">Guides</a></li>
-              <li><a href="/index.html#examples">Examples</a></li>
-              <li><a href="/native.html">Native</a></li>
-              <li><a href="/guide/contributors.html">Contributors</a></li>
-              <li><a href="/support.html">Support</a></li>
+              <li><a href="/gcdoc/index.html#quickstart">Quick Start</a></li>
```
Comment (bijoshtj) [inline html/index.html:40]: this will break if i clone the repo with some other name.

Comment (bijoshtj) [reply html/index.html:40]: same with all other places

**Pull request merged**